src/v2/view-builder/views/AutoRedirectView.js

```diff
@@ -34,6 +36,18 @@ const Body = BaseForm.extend({
 
     return titleString;
   },
+  showMessages() {
+    if (this.isUnlockSucess()) {
+      const container = '.o-form-error-container';
```

Comment:
Curious, do we have to use an error container for the success message?

Reply:
unfortunately yes, I tried with other containers and it didn't work

Reply:
Noted!